pom.xml

```diff
@@ -696,7 +696,6 @@
           </systemProperties>
           <reuseForks>true</reuseForks>
           <forkCount>${concurrency}</forkCount>
-          <argLine>-Djava.awt.headless=true ${argLine}</argLine>
```

Comment:
@jglick feel free to provide other working ways. But firstly ensure that jacoco would be usable.